lib/common/common.go

```diff
@@ -161,6 +161,20 @@ func GenerateManifest(layerData types.DockerImageData, dockerURL *types.ParsedDo
 			}
 			genManifest.App = app
 		}
+
+		for p, _ := range dockerConfig.Volumes {
+			n := filepath.Join("volume-", p)
+			sn, err := appctypes.SanitizeACName(n)
```

Comment:
Hrm maybe a corner case but don't we need to deal with duplicates? e.g. `/data#`, `/data$`